lib/src/util/prop_key_util.dart

```diff
@@ -20,7 +20,7 @@ import 'dart:collection';
 ///
 /// Example usage:
 ///
-///     var valuePropKey = getPropKey((props) => props.value, TextInput);
+///     var valuePropKey = getPropKey((props) => props.value, Dom.input);
```

Comment:
I think it'd be good to show a custom component here, since that's the main use-case